src/OpenAI/Platform/Azure.php

```diff
@@ -18,7 +18,7 @@ public function __construct(
         private string $baseUrl,
         private string $deployment,
         private string $apiVersion,
-        private string $key,
+        #[\SensitiveParameter] private string $apiKey,
```

Comment:
Needs to be adjusted in bundle config

Reply:
after the next tag the bundle needs some love anyways :+1:

Reply:
Prepared in
* https://github.com/php-llm/llm-chain-bundle/pull/38

Reply:
> after the next tag the bundle needs some love anyways 👍

Yes, lets do it in
* https://github.com/php-llm/llm-chain-bundle/pull/38